netlify.toml

```diff
@@ -55,6 +55,120 @@ from = "/knowledge-base/install/**"
 to = "/latest/software/"
 status = 302
 
+[[redirects]]
+# Moved KB article. See issue #749
+from = "/knowledge-base/guides/composition-revisions"
+to = "/latest/concepts/composition-revisions/"
+status = 301
```

Comment:
301 redirects sound like the right thing to do here, e.g. permanent redirect, but did want to call out that the previous pattern had been 302. Not sure if that divergence was intentional, but 301 does sound correct to me here.